src/py_converters_11.h

```diff
@@ -113,14 +177,14 @@ namespace PYBIND11_NAMESPACE { namespace detail {
                 return true;
             }
 
-            auto vertices = src.attr("vertices");
-            auto codes = src.attr("codes");
+            py::object vertices = src.attr("vertices");
+            py::object codes = src.attr("codes");
             auto should_simplify = src.attr("should_simplify").cast<bool>();
             auto simplify_threshold = src.attr("simplify_threshold").cast<double>();
 
-            if (!value.set(vertices.ptr(), codes.ptr(),
+            if (!value.set(vertices.inc_ref().ptr(), codes.inc_ref().ptr(),
```

Comment:
I don't know why this change is needed now, but otherwise things randomly crash.

Reply:
We think there are decrements in the iterator destructor there is a decr which we think we need to keep as there are still other usages of the iterators from c++ that we need to remove first.